stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -105,19 +105,82 @@ axes(A::AdjOrTransAbsMat) = reverse(axes(A.parent))
 IndexStyle(::Type{<:AdjOrTransAbsVec}) = IndexLinear()
 IndexStyle(::Type{<:AdjOrTransAbsMat}) = IndexCartesian()
 
+
 # MemoryLayout of transposed and adjoint matrices
-struct TransposeLayout{T} <: MemoryLayout{T} end
-struct CTransposeLayout{T} <: MemoryLayout{T} end
+abstract type ConjDenseColumns{T<:Complex} <: BlasMatrixLayout{T} end
```

Comment:
Wouldn't it be better to have a `ConjugatedLayout{T,L}` type as I suggested so that you don't have to define all of these `Conj*` types one by one?

Reply:
Yes. Sorry, I missed the suggestion.

The same argument may hold for ` UpperTriangularLayout `, `SymmetricLauout `, etc.